proto/ibc/lightclients/tendermint/v1/tendermint.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package ibc.lightclients.tendermint.v1;
 
-option go_package = "github.com/cosmos/ibc-go/v3/modules/light-clients/07-tendermint/types";
+option go_package = "github.com/cosmos/ibc-go/v3/modules/light-clients/07-tendermint;tendermint";
```

Comment:
This is key - note the `;tendermint` suffix in order to override the package name in the generated .pb.go files.

Otherwise the `tendermint.pb.go` file would have a package definition of `package _07_tendermint`